content/go/concepts/strings/terms/hasprefix/hasprefix.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'HasPrefix()'
+Description: 'Returns a boolean value indicating whether a given string begins with prefix.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Functions'
+  - 'Characters'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`HasPrefix()`** function returns a boolean value indicating whether a given string begins with prefix. The method returns `true` if the string begins with prefix, otherwise it returns `false`.
```

Comment:
```suggestion
The **`HasPrefix()`** function returns a boolean value indicating whether a given string begins with a given prefix. The method returns `true` if the string begins with the prefix, otherwise it returns `false`.
```